.vscode/tasks.json

```diff
@@ -0,0 +1,82 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+
+    // BaseTaskConfiguration
+    "presentation": {
+        "echo": true,
+        "reveal": "always",
+        "focus": true,
+        "panel": "shared",
+        "showReuseMessage": true,
+        "clear": true
+    },
+    "problemMatcher": {
+        "base": "$gcc",
+    },
+
+    // TaskConfiguration
+    "tasks": [
+        {
+            "label": "CLEAN",
+            "detail": "Clean output files of single effect ________________________ (Shift+Cmd+C)",
+            "type": "shell",
+            "command": "make clean -C ${workspaceFolder}/effects/${config:effect}",
+            "group": "build"
+        },
+        {
+            "label": "CLEAN ALL",
+            "detail": "Clean all output files ______________________ (Shift+Cmd+A, Shift+Cmd+C)",
+            "type": "shell",
+            "command": "make clean -C ${workspaceFolder}",
+            "group": "build"
+        },
+        {
+            "label": "BUILD",
+            "detail": "Build single effect ________________________________________ (Shift+Cmd+B)",
+            "type": "shell",
+            "command": "make -C ${workspaceFolder}/effects/${config:effect}",
+            "group":{
+                "kind": "build",
+                "isDefault": true
+            }
+        },
+        {
+            "label": "BUILD ALL",
+            "detail": "Build all effects ___________________________ (Shift+Cmd+A, Shift+Cmd+B)",
+            "type": "shell",
+            "command": "make -C ${workspaceFolder}",
+            "group": "build"
+        },
+        {
+            "label": "RUN",
+            "detail": "Run the effect without debugging ________________________ (Shift+Cmd+R)",
+            "type": "shell",
+            "command": "make run -C ${workspaceFolder}/effects/${config:effect}",
+            "group": "none"
+        },
+        {
+            "label": "DEBUG",
+            "detail": "Start debugging the effect _______________________________ (Shift+Cmd+D)",
```

Comment:
No, it launches VS Code integrated debugger which loads .vscode/launch.json configuration file and based on that executes "gdbserver" task as a pre-launch task and then attaches VS Code integrated debugger to this gdbserver.

This task is equal and does exactly the same as "Run -> Start debugging" menu item. But since "Run -> Start without debugging" menu item (i.e. Run) doesn't simply run the effect but also attaches debugger (i.e. this option is still not properly handled by C/C++ extension) I have put them both on a task list for convenience, i.e. properly working "Debug... (in VS Code debugger)" and "Run...(without attaching VS Code debugger)", so pressing Shift+Cmd+T will display it among other build system related tasks.